base/src/main/java/com/smartdevicelink/proxy/rpc/ShowConstantTbt.java

```diff
@@ -33,6 +33,7 @@
 
 import com.smartdevicelink.protocol.enums.FunctionID;
 import com.smartdevicelink.proxy.RPCRequest;
+import com.smartdevicelink.util.SdlDataTypeConverter;
```

Comment:
@kostyaBoss I would remove this import on line 36 `import com.smartdevicelink.util.SdlDataTypeConverter;`.

(The idea is to just make the description changes and nothing else)